### ISSUE TITLE
Darken the orange color to make white text more readable [ FEAT REQ ]

### ISSUE THREAD

**Issue by Hew-ux** [octoshrimpy/quik#129]
**Is your feature request related to a problem? If so, please describe the problem.**
I'd love to use the "Automatic contact colors" option, however I find it difficult to read the text when it selects the lighter orange color as the text background.
![Screenshot_20240710-185539~3](https://github.com/octoshrimpy/quik/assets/67527064/50c3a3fb-705a-4564-8190-08f6147288a9)

**Describe the feature you'd like**
To make the orange color slightly darker, or less saturated.

**Comment by lascapi** [comment]
Or use a very dark color for the text.

**Comment by accellarando** [comment]
qksms has some open tickets that may be relevant, I've been running into contrast issues too.

https://github.com/moezbhatti/qksms/issues/1564

https://github.com/moezbhatti/qksms/issues/1920

**Comment by octoshrimpy** [comment]
Agreed. The contact colors look great against a darker bg, but need to be checked against color readability scores for sure.

**Comment by Careless** [comment]
why not put a toggle on the color selection page to switch between white or black text so that the user can decide how SMS text is displayed if the default contrast is not suitable for them.

Pulse SMS seems to be fairly decent at selecting white or black text depending on overall brightness of the 3 RGB values (i think).

**Comment by 4jNsY6fCVqZv** [comment]
> but text color needs to be checked against bubble color readability scores for sure.

@octoshrimpy I was able to find the following values for the speech bubbles with the “Automatic contact colors” setting.

![Contrast 7](https://github.com/user-attachments/assets/a11ef51d-5b6d-46ff-99a4-2655002665db)
![Contrast 6](https://github.com/user-attachments/assets/2f9d78d7-d9ef-4e01-bdcb-a5321f61e6d2)
![Contrast 5](https://github.com/user-attachments/assets/43a6e9cc-9761-4584-addc-1054df6ad608)
![Contrast 4](https://github.com/user-attachments/assets/c178b40d-0916-4275-91b4-9ba5a1eb4325)
![Contrast 3](https://github.com/user-attachments/assets/283463ea-80c6-4b1e-9c2f-0002a54ba7ba)
![Contrast 2](https://github.com/user-attachments/assets/4757679b-ab8f-4c6c-9bba-ac282de3c74f)
![Contrast 1](https://github.com/user-attachments/assets/b0ab3b13-cbb5-4c3e-a39a-33c653b8101f)

My system is in dark mode.
The font size in QUIK is set to “Normal”.
I am using version 4.0.7 from F-Droid.

**Comment by Careless** [comment]
> > but text color needs to be checked against bubble color readability scores for sure.
> 
> @octoshrimpy I was able to find the following values for the speech bubbles with the “Automatic contact colors” setting.
> 

> My system is in dark mode. The font size in QUIK is set to “Normal”. I am using version 4.0.7 from F-Droid.

and what are we supposed to do with this information?

**Comment by 4jNsY6fCVqZv** [comment]
> and what are we supposed to do with this information?

I think the version concerned is basically good to know for an issue or contributions to it.

As far as I know, the font size has an effect on legibility.

And with the dark mode, I wanted to say that I did the contrast check with the dark theme of QUIK.

**Comment by 4jNsY6fCVqZv** [comment]
If you are referring to the values found, I would suggest adjusting the color selection for automatic contacts in QUIK.

**Comment by 4jNsY6fCVqZv** [comment]
I would also suggest to adjust the issue title:
“Automatic contact colors should be fully accessible.”
For example, in line with the W3C Accessibility Guidelines.

**Comment by 4jNsY6fCVqZv** [comment]
The corresponding place in the code is from here onwards https://github.com/octoshrimpy/quik/blob/1ba78533036634a40ac65ae8c95abebf3950f921/presentation/src/main/res/values/colors.xml#L68, isn't it?

I can imagine to work on a solution.

**Comment by 4jNsY6fCVqZv** [comment]
@octoshrimpy The relevant code is https://github.com/octoshrimpy/quik/blob/1ba78533036634a40ac65ae8c95abebf3950f921/presentation/src/main/res/values/colors.xml#L68-L76, isn't it?

I can imagine to work on a solution.

**Comment by 4jNsY6fCVqZv** [comment]
@octoshrimpy Is this where the relevant code for such an improvement can be found?

https://github.com/octoshrimpy/quik/blob/1ba78533036634a40ac65ae8c95abebf3950f921/presentation/src/main/res/values/colors.xml#L68-L76

Or are there other parts of the code that should be looked at in this context?
